index.html

```diff
@@ -10,392 +10,422 @@
-          <p class="text -small -color-dark">Could texts be used not only to send reminders but also to build an ongoing two-way dialogue with a trusted health partner? Would this conversation help improve medication adherence?</p>
+        <div class="content-wrap">
+          <!--Question heading-->
+          <div class="heading-container">
+            <h2 class="text -x-small -color-dark -caps -wide-spacing -bold">The Question</h2>
+            <h3 class="text -large -serif -color-dark">How can technology help patients deal with these factors so they adhere more accurately and consistently?</h3>
+          </div>
+          <!--Question content-->
+          <div class="question__content narrow-padding">
+            <div class="phone"></div>
+            <p class="text -small -color-dark">
+            Access was our primary concern. Smartphone adoption is growing but 35% of the U.S. population still doesn’t own one.<cite><a href="">4</a></cite> Reliance on personal computers is also problematic, especially among disadvantaged patients.</p>
+            <p class="text -small -color-dark">We narrowed our exploration to technology that was pervasive and familiar. Nearly 9 out of 10 American adults use a phone capable of texting.<cite><a href="">5</a></cite> Furthermore, adherence studies using text reminders show promising results.<cite><a href="">6</a></cite> Texting shows the potential to address difficult issues around the world and across verticals. In Libya, the development of the world’s first SMS voter registration system led to 1.5 million voter registrations.<cite><a href="">7</a></cite></p>
+            <p class="text -small -color-dark">Could texts be used not only to send reminders but also to build an ongoing two-way dialogue with a trusted health partner? Would this conversation help improve medication adherence?</p>
+          </div>
         </div>
       </section>
+
     </div>
-  </div>
-  <!--1) GREEN BACKGROUND END-->
+    <!--1) GREEN BACKGROUND END-->
 
 
-  <!--2) BLUE BACKGROUND START-->
+    <!--2) BLUE BACKGROUND START-->
     <div class="bg-blue">
+
       <!--MEET FLO SECTION-->
-      <div class="content-wrap">
-        <div class="flo-img"></div>
-        <section class="meet-flo">
-          <div class="centered">
-            <p class="text -x-small -color-white -caps -wide-spacing -bold">The Concept</p>
-            <p class="text -large -serif -color-white">Meet Florence</p>
+      <section class="meet-flo">
+        <div class="content-wrap">
+          <!--Meet Flo heading-->
+          <div class="heading-container">
+            <h2 class="text -x-small -color-white -caps -wide-spacing -bold">The Concept</h2>
+            <h3 class="text -large -serif -color-white">Meet Florence</h3>
```

Comment:
Same here, re: heading. "Meet Florence" isn't subdividing content under "The Concept". even though it's much larger, it's probably okay just as a `<p>`.